packages/launcher-component/src/loaders/buildimage-loader.tsx

```diff
@@ -41,7 +41,9 @@ const convertToPairs = object => {
 export const convertToObject = (vars: Array<{ key: string; value: string }>) => {
   const result = {};
   for (const env of vars) {
-    result[env.key] = env.value;
```

Comment:
shouldn't we merge the other one (2d array) before fixing this 🤔😁

Reply:
resolved by rebasing ;)